public/app/plugins/datasource/prometheus/components/PromLink.tsx

```diff
@@ -44,13 +44,22 @@ const PromLink: FC<Props> = ({ panelData, query, datasource }) => {
           scopedVars: enrichedScopedVars,
         } as DataQueryRequest<PromQuery>;
 
+        const customQuery: any = {};
```

Comment:
```suggestion
        const customQueryParameters: { [key: string]: string }  = {};
```

I think the name variable should be `customQueryParameters`, so it is more clear what it is and we can more specific also have type here.